sql/mob_spawn_points.sql

```diff
@@ -58242,11 +58247,11 @@ INSERT INTO `mob_spawn_points` VALUES (17658244,'Hannequet','Hannequet',10200,46
 INSERT INTO `mob_spawn_points` VALUES (17658245,'Hannequet','Hannequet',10200,466.629,19.987,-32.151,40);
 INSERT INTO `mob_spawn_points` VALUES (17658246,'Hannequet','Hannequet',10200,472.575,19.572,-39.897,70);
 INSERT INTO `mob_spawn_points` VALUES (17658247,'Ephemeral_Murex','Ephemeral Murex',10193,439.16,20.968,21.467,100);
-INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',10230,0,0,0,130);
-INSERT INTO `mob_spawn_points` VALUES (17658249,'Tejas','Tejas',10230,0,0,0,160);
-INSERT INTO `mob_spawn_points` VALUES (17658250,'Tejas','Tejas',10230,0,0,0,190);
-INSERT INTO `mob_spawn_points` VALUES (17658251,'Tejas','Tejas',10230,0,0,0,220);
-INSERT INTO `mob_spawn_points` VALUES (17658252,'Tejas','Tejas',10230,0,0,0,250);
+INSERT INTO `mob_spawn_points` VALUES (17658248,'Tejas','Tejas',0,-544,-3,114,12);
```

Comment:
i can sort out all packets for those mobs to check pos (i kept all of them)
[02:05:10] 0x0E, 68 Bytes - (Update) Entity
   0  1  2  3  4  5  6  7  8  9  A  B  C  D  E  F
   -----------------------------------------------
   0E 22 BC 07 9B 71 0D 01 9B 01 0F 42 D1 42 08 C4    ."¼.›q..›..BÑB.Ä
   5E BA 49 C1 1D 5A 90 C2 0C 10 00 00 28 28 00 03    ^ºIÁ.ZÂ....((..
   07 9D 00 00 00 27 00 20 20 00 00 00 00 00 00 00    ....'.  .......
   00 00 A4 01 53 76 61 72 62 68 61 6E 75 00 00 00    ..¤.Svarbhanu...
   00 00 00 00                                        

but the pos i added back then where those from infobar so while mob was moving... i think the one from packet give the first pos received (whatever it is even if i did not encountered the mob myself)